modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -1361,6 +1364,33 @@ sub _summarise_compara_db {
   $dbh->disconnect;
 }
 
+sub _intraspecies_sql {
+  return qq(
+    select mls.species_set_id, mls.method_link_species_set_id, count(*) as count
+      from method_link_species_set as mls, 
+        method_link as ml, species_set as ss, genome_db as gd 
+      where mls.species_set_id = ss.species_set_id
+        and ss.genome_db_id = gd.genome_db_id 
+        and mls.method_link_id = ml.method_link_id
+        and ml.class = "GenomicAlignBlock.pairwise_alignment"
+      group by mls.method_link_species_set_id, mls.method_link_id
+      having count = 1
+  );
+}
+
+sub _homologies_sql {
+  return qq(
+    select ml.type, gd.name, gd.name, count(*) as count
```

Comment:
there is double gd.name in the select statement, i know it was just moving the code but i am not sure if we need gd.name twice, might be good to remove one of it unless it is supposed to be another column name.

Reply:
Yes, it's necessary. The code is combining two arrayrefs before it builds the hash - see line 1347. If we didn't have the name twice, the paralogues hash would be incomplete.